gravitl/netmaker-ui
# UI enhancements/fixes

TESTING STEPS
1. Go to node details, check tht postup/down are not visible
2. Go to edit host. Enter an invalid erbosity, check that validation msg says valid range is 0-4
3. Go to a host, under the networks tab. Check that the addresses of the connected nodes show, instead of local address
